sci-chemistry/modeller/modeller-10.5.ebuild

```diff
@@ -20,7 +20,8 @@ IUSE="doc examples"
 
 RESTRICT="mirror"
 
-DEPEND=">=dev-lang/swig-1.3"
+DEPEND=">=dev-lang/swig-1.3
```

Comment:
swig 1.3 is so ancient that you can just make this <...

Reply:
also, let's make it a BDEPEND while here given it has to be executed during the build

Reply:
Also, for adding upper-bound restrictions, please:
a) include a comment above the *DEPEND block mentioning the relevant Gentoo bug ("<swig-4.3.0 for bug #1234")
b) ensure there's an upstream bug to refer to, or it's going to be there forever

Reply:
modeller is not open source, and has no obvious way to report issues except the mailing list. I'll make a note of the gentoo bug,

Reply:
OK, please put that in the commit message.

Reply:
done